tests/notebooks/linear_decoder.ipynb

```diff
@@ -23,17 +23,9 @@
    "cell_type": "code",
```

Comment:
<p>We should give credit here to Valentine. The best way to me seems to cite their paper :)</p><p>https://www.biorxiv.org/content/biorxiv/early/2019/08/16/737601.full.pdf</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/YosefLab/scVI/pull/497/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
I'll add to the readme instead.